SalesforceCommerceCloud/pwa-kit
# @W-13578461@ GitHub actions for hybrid testing

This PR contains changes to PWA github actions to allow better reusability when executed in a different Github action repository.  These changes are needed to support the CI running on [pwa-sfra-hybrid-tests](https://github.com/SalesforceCommerceCloud/pwa-sfra-hybrid-tests)

This contains the changes from #1466 as well as a fix to the `--wait` flag on `npm run push`

Testing the Github actions
* Run the CI and verify all tests pass

Testing generated hybrid test project (see #1466)
* `node packages/pwa-kit-create-app/scripts/create-mobify-app-dev.js --preset retail-react-app-hybrid-test-project`
* Verify that an app is generated
* Build and run the generated app. Verify that it works. Verify that trying to navigate to /cart or /checkout triggers redirects (will return a page not found unless you have hybrid SFRA / ecdn / composable-dev-server set up)
* Also run the generator without this preset
* Verify that the other extensible retail react app presets are not affected by this change (check router.jsx)

Testing `--wait`
* `npm run push -- -s your-target-project-id --message test --target your-target-env-id --wait`
